docs/config/@config.md

```diff
@@ -180,7 +186,7 @@ This will force the world to be restarted if the config is changed in the mod op
 
 Sub Categories
 --------------
-If you want Sub Categories, you must specify them. A good way to do this is to make a private inner class, and create a singleton instance in it's parent. 
+A good way to male a `Sub Category` is to make a private inner class, and create a singleton instance in it's parent. 
```

Comment:
male -> make

Why is "Sub Category" in a code block?

Honestly I'm not quite sure how I feel about this sentence as a whole. It seems more like a personal preference, since it can just be any object type.

Reply:
I see your point, i'll change it real quick.... Thank god you guys are so good at spotting mistakes... i cant spot my mistakes ever.